src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Admins/Queries/Lookup/LookupAdminMemberRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using MediatR;
+
+namespace SFA.DAS.ApprenticeAan.Application.Admins.Queries.Lookup;
+public class LookupAdminMemberRequest : IRequest<LookupAdminMemberResult?>
+{
+    public string Email { get; set; } = null!;
+    public string FirstName { get; set; } = null!;
+    public string LastName { get; set; } = null!;
```

Comment:
These fields are not required by the handler. I see why they are in here, I guess you need to use a model to take the incoming request body and then pass only email here.